casper/src/main/scala/coop/rchain/casper/engine/NodeSyncing.scala

```diff
@@ -116,7 +119,8 @@ class NodeSyncing[F[_]
     case b: BlockMessage =>
       Log[F]
         .info(s"BlockMessage received ${PrettyPrinter.buildString(b, short = true)} from $peer.") *>
-        blockMessageQueue.enqueue1(b)
+        blockMessageQueue.enqueue1(b) *>
+        incomingBlocks.enqueue1(b.blockHash)
```

Comment:
Why is this needed? When node is in LFS syncing mode it will receive blocks with LfsBlockRequester and this blockMessageQueue is just for that. After LFS sync is done NodeRunning is responsible to handle network mesages.

Reply:
Please take a look at [commit](https://github.com/rchain/rchain/pull/3650/commits/20171946cdf4b0e439d3808b51ee8d059b1d1df1). This `blockProcessorQueue` fills only when node running. Are we consider that LFS blocks should not be handled with BlockReceiver?